packages/Card/src/index.d.ts

```diff
@@ -64,3 +84,22 @@ declare namespace Card {
     }
   }
 }
+declare namespace Footer {
+  namespace types {
+    namespace themes {
+      const BLACK: any;
+      const BLUE: any;
+      const GREEN: any;
+      const GREY: any;
+      const LIGHT_BLUE: any;
+      const LIGHT_ORANGE: any;
+      const ORANGE: any;
+      const severityThemes: any;
+      const NO_RISK: any;
+      const LOW_RISK: any;
+      const MEDIUM_RISK: any;
+      const HIGH_RISK: any;
+      const ALERT: any;
+    }
+  }
+}
```

Comment:
This whole `index.d.ts` file can be updated again, probably by running the `yarn generateTypeDefinitionsForJS` script from the root of the Paprika repo.